userguide/subscription/includes/glossary.adoc

```diff
@@ -1,17 +1,17 @@
 
-*Catalog* - Catalogs describes the set of Products, Plans, Price Lists and Rules that are being made availabel to customers. A catalog lets Kill Bill know what is being offered for sale and how much to charge for it.
+*Catalog* - A description of the set of Products, Plans, Price Lists and Rules that are being made available to customers. A catalog lets Kill Bill know what is being offered for sale and how much to charge for it.
 
-*Payment Method* - Payment methods store the details required for Kill Bill to trigger a payment. They represent things like credit cards, debit cards, or Paypal accounts. Each account can have many Payment Methods and at most one Default Payment Method. Note that much of the representation of a Payment Method is plugin-specific and is actually stored by the Payment Plugin itself.
+*Payment Method* - A record of the details required for Kill Bill to trigger a payment. Payment Methods represent things like credit cards, debit cards, or PayPal accounts. Each Account can have many Payment Methods and at most one Default Payment Method. Note that much of the representation of a Payment Method is plugin-specific and is actually stored by the Payment Plugin itself.
 
-*Plan* - Plans are Catalog objects that define how a Product is to be paid for. They can have multiple phases (e.g. trial, discount, evergreen). The phase specifies the frequency of payment and the prices in multiple currencies. Each phase can have fixed and recurring prices associated with it. Each Product can have multiple Plans specifying different ways that the Product can be purchased (e.g. a monthly Plan and an annual Plan)
+*Plan* - A Catalog object that defines how a Product is to be paid for. Plans can have multiple phases (e.g. trial, discount, evergreen). The phase specifies the frequency of payment and the prices in multiple currencies. Each phase can have fixed and recurring prices associated with it. Each Product can have multiple Plans specifying different ways that the Product can be purchased (e.g. a monthly Plan and an annual Plan).
 
-*Price list* - Price lists are collections of plans that might be associated with a particular program (e.g. an affiliate program or a discount program). A plan can be in many price lists. There must be a default price list that is used when no pricelist is specified. When a subscription changes plans, Catalog rules can be used to determine which Price list the new Plan should use.
+*Price list* - A collection of Plans that might be associated with a particular program (e.g. an affiliate program or a discount program). A Plan can be in many Price Lists. There must be a default Price List that is used when no Price List is specified. When a Subscription changes Plans, Catalog Rules can be used to determine which Price List the new Plan should use.
```

Comment:
> A Plan can be in many Price Lists.

I realize the doc was already saying that, but I am wondering if this is really true. Would you be able to make an experiment and try it out?

Reply:
I'm not sure, but I will try to check on this.

Reply:
I have thought awhile about this. It seems reasonable enough that one price list could contain some of the same plans as another one. I created a test catalog that included this case and it passed the validation tool. I don't know any other way to check it.

Reply:
OK, good enough :-)